client/client.go

```diff
@@ -682,7 +682,7 @@ func (r *NotaryRepository) bootstrapRepo() error {
 	logrus.Debugf("Loading trusted collection.")
 
 	for _, role := range data.BaseRoles {
-		jsonBytes, err := r.fileStore.GetMeta(role, store.NoSizeLimit)
+		jsonBytes, err := r.fileStore.GetSized(role, store.NoSizeLimit)
```

Comment:
we can do this in a separate PR, but should we phase out `store.NoSizeLimit` and change these calls to `Get()` instead of `GetSized()`?